docs/userguide/async.md

```diff
@@ -0,0 +1,74 @@
+# Asynchronous Workflows
+
+Asynchronous workflows describe situations where the loop between measurements and
+recommendations is more complex and needs to incorporate various other aspects. These
+could for instance be:
+- **Distributed workflows**: When recommendations are distributed across several
+  operators, e.g. at different locations or in several reactors, some experiments might
+  have been started, but are not ready when the next recommendation is requested.
+  Without further consideration, the algorithm would be likely to recommend the pending
+  measurements again, as it is unaware they were already started.
+- **Partial targets**: Relevant when dealing with multiple targets which require very
+  different amounts of time to measure. When a new recommendation is requested, the
+  targets of previously recommended points might only partially available, but that
+  info should ideally already be considered.
+
+
+## Pending Measurements
+
+With *pending measurements* we mean recommendations whose measurement process has
```

Comment:
I think this paragraph needs to a slight overhaul. The problem I currently see: Above, you nicely describe the two *very different* situations that can arise (but also somehow imply that there might be others, since you say "for instance" – do we actually want to keep this or make it stricter and specifically state that these are the two classes of scenarios?)

But now it becomes a bit unclear what the "pending measurements" approach actually solves. Correct me if I'm wrong, but I think we only address the "distributed workflows" class, while the other remains a problem, right? If so, we should clearly state is by saying that BayBE offers a solution to the former via "pending_measurements", not to the latter? But somehow you seem to imply that it also solves the second, since you say `A measurement is pending whenever one or more of its targets are not measured yet`.... I'm a bit confused, tbh

Reply:
Regarding the first point: There might be more scenarios, right? So I'd keep the "for instance".

Reply:
I think we just need a very clear definition of what `pending` means. In my interpretation, this would be `A point is called *pending* if the process of measuring the point has started, but is not finished before a subsequent recommendation is requested`. This should fit both cases, and we can then write that we support adding pending points in Distributed Workflow setting resp. in a "Level 2" setting. Maybe these levels should also be explicitly mentioned here to make this more clear?

Reply:
Userguide was reworked please reevaluate

Reply:
added my comments, so all good from my end. Thanks for reworking, btw!